src/UKHO.ERPFacade.Common/IO/AzureTableReaderWriter.cs

```diff
@@ -74,6 +76,24 @@ await foreach (var entity in entities)
             return records.FirstOrDefault();
         }
 
+        public async Task UpdateEntity(string traceId, string updateColumn)
```

Comment:
Need unit tests for this method

Reply:
Done. This method is separated for UpdateRequestTime & UpdateResponseTime.